### ISSUE TITLE
Need Standardized Function Comments

### ISSUE THREAD

**Issue by MatthewChastain** [smacademic/project-cgkm#60]
Currently there is no standard format for function comments in ArcPlanner's source code. We should have all function comments include parameters, returns, and a function description.

**Comment by KevinKelly25** [comment]
In order to avoid many merge conflicts, which is a high possibility with this type of issue. I believe this issue should be resolved at the very end of the milestone right before release when all other PRs are closed. 

However, I do think we should discuss a format beforehand. My suggestion is to use the following format, taken from our `arc.dart` file: 
```
  /*
  * Description: The parameters supply needed information to create a Task within
  *  the SQLite database with the exception of the ID. This ID needs to be 
  *  generated with a UUID. This data is sent to the SQLite database to be
  *  inserted into the task table. If successfully inserted the data should be
  *  used to create a task object. That task object should then be added to the
  *  list of tasks belonging to the Arc which called the instance method.
  * @param title represents the title of the Task
  * @param description is the information that describes the  task objective. 
  *  This is null by default
  * @param dueDate is the date when the task should be completed by.
  *  This is null by default
  * @param location is where the task takes place. This is null by default
  * Return: Returns the created task
  */

This would conform to the decision we made in #37

**Comment by jrm86** [comment]
I've created a [Wiki page](https://github.com/smacademic/project-cgkm/wiki/Standardized-file-header-and-function-comments) so that we can view and edit function comment format without risking merge conflicts. Dart actually has a [standard](https://www.dartlang.org/guides/language/effective-dart/documentation#do-use-square-brackets-in-doc-comments-to-refer-to-in-scope-identifiers) for function comments which enables dartdoc (kinda like javadoc).

**Comment by KevinKelly25** [comment]
Closed with PR #106
